src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+class RightTriangle extends Figure {
```

Comment:
The class 'RightTriangle' should not extend 'Figure' directly if 'Figure' is an abstract class meant for providing common fields. Consider using an interface to define common methods if 'Figure' is not meant to provide any default implementations.